src/pages/keyblocks/[id].vue

```diff
@@ -37,6 +37,8 @@ const { keyblockDetails } = storeToRefs(keyblockDetailsStore)
 const { fetchKeyblock } = keyblockDetailsStore
 const route = useRoute()
 
+const showTabs = computed(() => keyblockDetails.value && !keyblockDetails.value?.notExistent)
```

Comment:
The variable returning boolean should be `is` prefixed. Now it looks like function name on the first sight. You may accept Michelle's edit

Reply:
Yup, done